### DISCUSSION TITLE
Layout PDF

### DISCUSSION THREAD

**Discussion by sidneiJr1984** [DenysXavier/WSBoletoSantander#60] [General]
Boa tarde,

Muito legal sua classe, bem produtiva e facil de usar. Por acaso você tem também o gerador do PDF do boleto?

**Comment by DenysXavier** [comment]
Olá, @sidneiJr1984!

Infelizmente, não.

Onde eu trabalhava, usávamos uma versão antiga do PHP Boleto com adaptações para registro com essa biblioteca (WSBoletoSantander) e para geração do PDF com o html2pdf (veja  [spipu/html2pdf](https://github.com/spipu/html2pdf)).

Lamento.
